config/mailinglist/rules/jdk.json

```diff
@@ -113,83 +113,24 @@
             "test/make"
         ],
         "compiler-dev": [
-            "make/Compile",
-            "make/data/symbols/java.compiler",
+            "make/CompileInterimLangtools",
+            "make/data/symbols/",
             "make/jdk/src/classes/build/tools/depend",
             "make/jdk/src/classes/build/tools/taglet/Preview.java",
             "make/langtools/",
             "make/modules/jdk.compiler/",
             "make/scripts/generate-symbol-data.sh",
-            "src/hotspot/share/classfile/classFileParser.hpp",
-            "src/hotspot/share/code/dependencies.cpp",
-            "src/hotspot/share/interpreter/bytecode.cpp",
-            "src/hotspot/share/interpreter/bytecodeTracer.cpp",
-            "src/hotspot/share/interpreter/linkResolver.hpp",
-            "src/hotspot/share/interpreter/templateTable.cpp",
-            "src/hotspot/share/interpreter/templateTable.hpp",
-            "src/hotspot/share/runtime/reflection.hpp",
-            "src/java.base/share/classes/java/lang/",
             "src/java.base/share/classes/jdk/internal/PreviewFeature.java",
```

Comment:
I think this shows the question on maintenance from the jdk-dev list - the (internal) PreviewFeature may be moved to a different (internal) place sometime in the foreseeable future. Who will be updating this list?

Reply:
When in doubt, I think it's better to err on the side of caution, so perhaps it should not be included in the list at all. But will expand on this in the jdk-dev thread.